sshd/issue_invite.sh

```
@@ -10,86 +10,97 @@ set -e
 INVITE_CODE=
 USERNAME=getter
 AUTOMATED=false
+KEY=
 
 function usage () {
-  echo "$0 [-u username] [-i invite code] [-a]"
+  echo "$0 [-u username] [-i invite code] [-a] [-k key]"
   echo "  -i: invite code (if unspecified, a new invite code is generated)"
   echo "  -u: username (default: getter)"
   echo "  -a: do not output complete invite URL"
+  echo "  -k: public key (if unspecified, a new key is generated)"
   echo "  -h, -?: this help message"
   exit 1
 }
 
-while getopts i:u:ah? opt; do
+while getopts i:u:k:ah? opt; do
   case $opt in
     i) INVITE_CODE="$OPTARG" ;;
     u) USERNAME="$OPTARG" ;;
     a) AUTOMATED=true ;;
+    k) KEY="$OPTARG" ;;
```

Let's throw an error when `-i` and `-k` are both specified.